packages/venia-concept/src/components/MiniCart/miniCart.js

```diff
@@ -95,11 +96,30 @@ class MiniCart extends Component {
                         <Icon name="x" />
                     </Trigger>
                 </div>
-                <div className={classes.body}>{productList}</div>
-                <div className={classes.footer}>
-                    <div className={classes.summary}>{totalsSummary}</div>
-                </div>
-                <Checkout cart={cart} />
+                {cart.details.items && cart.details.items.length ? (
+                    <Fragment>
+                        <div className={classes.body}>{productList}</div>
+                        <div className={classes.footer}>
+                            <div className={classes.summary}>
+                                {totalsSummary}
+                            </div>
+                        </div>
+                        <Checkout cart={cart} />
+                    </Fragment>
+                ) : (
+                    <div className={classes.empty}>
+                        <h3 className={classes.emptyTitle}>
+                            <span>
```

Comment:
Why do we need `span` here?

Reply:
I just noticed pretty almost all titles have this `span` so just wanted to keep it consistent, but yes it can be removed.